examples/README.md

```diff
@@ -13,7 +13,7 @@
   
   First, make sure to have hax installed in PATH. Then:
   
-  * With Nix, `nix develop .#fstar-examples` setups a shell automatically for you.
+  * With Nix, `nix develop .#fstar` setups a shell automatically for you.
      
   * Without Nix:
     1. install F* `v2023.09.03` manually (see https://github.com/FStarLang/FStar/blob/master/INSTALL.md);
```

Comment:
When you update F*, please do it everywhere. Looks like the regular docs aren't updated. Not sure if there are more places.

Reply:
Arf, sorry, I missed that completely... that's fixed.